testing/rkt/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Brian Rosner <brian@brosner.com>
+# Maintainer: Brian Rosner <brian@brosner.com>
+pkgname=rkt
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="container engine for Linux designed to be composable, secure, and built on standards"
+url="https://github.com/coreos/rkt"
+arch="x86_64"
+license="ASL 2.0"
+depends=""
+depends_dev=""
+makedepends="$depends_dev coreutils grep gzip diffutils cpio go bc linux-headers wget file gnupg autoconf automake bash openssl-dev acl-dev xz sed"
+subpackages=""
+source="rkt-$pkgver.tar.gz::https://github.com/coreos/rkt/archive/v$pkgver.tar.gz"
+builddir="$srcdir/rkt-$pkgver"
+
+build() {
+	cd "$builddir"
+	./autogen.sh
```

Comment:
Unfortunately APKBUILD does not run with `set -e` (yet, it’s on our TODO list to change it), so every statement that may fail, but should not fail, and is not the last statement of the function, should end with `|| return 1`.